src/.cargo/config.toml

```diff
@@ -1,8 +1,11 @@
 [target.thumbv7em-none-eabihf]
```

Comment:
This file should be removed, right? It will clash with #308

Reply:
If I'm not mistaken this git showing the deletion of `.cargo/config` as a "rename", it's confused :) I think when #308 is merged this may change. In anycase there is no `src/06-hello-world/.cargo/config` in my version of `src/06-hello-world`.

Reply:
Hmm, GitHub is showing a merge conflict with the `src/06-hello-world/.cargo/config` file. Could you rebase this to master?